lms/djangoapps/course_home_api/outline/v1/views.py

```diff
@@ -177,6 +220,25 @@ def get(self, request, *args, **kwargs):
 
         return Response(serializer.data)
 
+    @staticmethod
+    def get_start_block(blocks, block):
+        children = blocks.get_children(block)
+        if not children:
+            return None
+
+        return children[0]
+
+    @staticmethod
+    def get_resume_block(blocks, block):
+        children = blocks.get_children(block)
+        if not children:
+            return None
+
+        for child in children:
+            if blocks.get_xblock_field(child, 'resume_block', False):
+                return child
+        return None
```

Comment:
Nit: unnecessary return None

Reply:
I personally think it's a little clearer to not rely on the implicit None return when you aren't a `void` method. But tastes vary.

Reply:
Yeah, that's fair.